pkg/registry/apps/application/rest.go

```diff
@@ -66,6 +67,13 @@ const (
 	AnnotationPrefix = "apps.cozystack.io-"
 )
 
+// Application label keys
+const (
+	ApplicationKindLabel  = "apps.cozystack.io/application.kind"
+	ApplicationGroupLabel = "apps.cozystack.io/application.group"
+	ApplicationNameLabel  = "apps.cozystack.io/application.name"
+)
```

Comment:
This should be part of the API:
```suggestion
const (
	ApplicationKindLabel  = appsv1alpha1.ApplicationKindLabel
	ApplicationGroupLabel = appsv1alpha1.ApplicationGroupLabel
	ApplicationNameLabel  = appsv1alpha1.ApplicationNameLabel
)
```
and corresponding constants should be added to `pkg/apis/apps/v1alpha1/types.go`.